auth2.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+function signup2()
+{
+	if (count($_POST) > 0) {
+		$fh = fopen('users.csv', 'a+');
+		fputs($fh, $_POST['email'] . ';' . password_hash($_POST['password'], PASSWORD_DEFAULT) . PHP_EOL);
+		fclose($fh);
+		echo "You created your account. Sign in please.";
+		return;
+	}
+
+	// echo '<form method="POST">
+	// 	<input type="email" name="email" placedholder="Email" />
+	// 	<input type="password" name="password" placedholder="password" />
+	// 	<input type="submit">Create Account</button>
+	// </form>';	
```

Comment:
Is this going to be used in the future? If not it should be deleted.

Reply:
Original auth2 is now removed and this should be gone.